src/_scss/pages/search/results/visualizations/geo/geoVisualization.scss

```diff
@@ -1,35 +1,171 @@
 .results-visualization-geo-section {
-    position: relative;
-    @import "./_topSection";
+  position: relative;
+  @import "./_topSection";
 
-    h2.visualization-title {
-        @include h3;
-        margin: 0;
-    }
+  h2.visualization-title {
+    @include h3;
+    margin: 0;
+  }
 
-    .map-container {
-        height: rem(475);
-        width: 100%;
-    }
-    @import "./map/map";
-    .visualization-tooltip {
-        @import "components/visualizations/tooltip/_tooltip";
-    }
-    //HIDE THE MAPBOX COMPASS CONTROLS
-    & .mapboxgl-ctrl-compass {
-        display: none !important;
-    }
-    //Hide info button that was added after mapbox package update to v1.13.0
-    & .mapboxgl-ctrl-attrib-button {
-        display: none !important;
+  .map-container {
+    height: rem(475);
+    width: 100%;
+
+    .map__filters-toggle {
+      background-color: $color-white;
+      width: rem(16);
+      height: rem(35);
+      top: rem(24);
+      left: 23rem;
+      position: absolute;
+      z-index: 2;
+      border: solid 1px $gray-cool-10;
+      padding: rem(10) rem(4);
+      display: flex;
+      align-items: center;
+      border-radius: 0 4px 4px 0;
+
+      &.closed {
+        left: 0;
+        transition: left;
+      }
+
+      &.open {
+        left: 23rem;
+        transition: left;
+      }
+
+      svg {
+        width: rem(6);
+        height: rem(14);
+        color: $blue-50;
+      }
     }
-    & .mapboxgl-ctrl-attrib {
-        display: none !important;
+
+    .map__filters-container {
+      background-color: $color-white;
+      top: rem(8);
+      border-top-right-radius: 4px;
+      border-bottom-right-radius: 4px;
+      position: absolute;
+      width: 23rem;
+      z-index: 2;
+      transition: 1s;
+      opacity: 1;
+      border: solid 1px $gray-cool-10;
+      box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.1);
+      padding: rem(16) rem(48) rem(24) rem(24);
+
+      &.closed {
+        opacity: 0;
+        display: none;
+        transition: opacity 1s;
```

Comment:
These two transition properties should be combined into one line, using the transition shorthand, or they should use the more specific proprty name for each.
This way they the last one is overriding the first one.